dashboard/src/shared/types.ts

```diff
@@ -1,5 +1,9 @@
 import { JSONSchemaType } from "ajv";
 import { RouterState } from "connected-react-router";
+// TODO(minelson): Why is lint complaining:
+// error  Unable to resolve path to module 'rxjs'  import/no-unresolved
```

Comment:
I guess is something missing in the eslintrc.json (perhaps in `import/resolver` ?) not sure, I'd have to check

Reply:
So finally found the right incantation here, from https://stackoverflow.com/a/55280867 . It looks like it was because `rxjs` is itself written in typescript directly, but node was (?) only looking for js files in `node_modules`.